web/src/app/services/HeartbeatMonitorForm.tsx

```diff
@@ -47,13 +45,17 @@ export default function HeartbeatMonitorForm(
         <Grid item xs={12}>
           <FormField
             fullWidth
-            component={NumberField}
+            component={DurationField}
             required
-            type='number'
-            label='Timeout (minutes)'
+            label='Timeout'
```

Comment:
There's a `getByLabel()` test that looks for `'Timeout (minutes)'` that should be updated here:
https://github.com/target/goalert/blob/f4b027bae129745186bf01c0af6bde154d7d484f/test/integration/service.spec.ts#L65